test/functional/readconcern_tests.js

```diff
@@ -121,12 +186,13 @@ describe('ReadConcern', function() {
           client =
             test.urlOptions != null
               ? configuration.newClient(url, options)
-              : configuration.newClient({ w: 1 }, { poolSize: 1, readConcern: test.readConcern });
+              : configuration.newClient(url);
```

Comment:
Do we no longer need the `{ w: 1 }` and `{ poolSize: 1 }` options?

Reply:
I checked the same tests in the old file, and it seems to just create the client for these tests with either `newClient(url)` or `newClient(url, options)`